fission-cli/library/Fission/CLI/Types.hs

```diff
@@ -812,17 +814,15 @@ instance
   , Contains errs errs
   , MonadIPFSIgnore (FissionCLI errs cfg)
   )
-  => JWT.Resolver (FissionCLI errs cfg) where
+  => UCAN.Resolver (FissionCLI errs cfg) where
   resolve cid@(IPFS.CID hash') = do
     _ <- IPFS.Daemon.runDaemon
     IPFS.runLocal ["cat"] (Lazy.fromStrict $ encodeUtf8 hash') <&> \case
       Left errMsg ->
         Left $ CannotResolve cid (ConnectionError $ toException errMsg)
 
       Right (Lazy.toStrict -> resolvedBS) ->
-        case eitherDecodeStrict resolvedBS of
-          Left  _   -> Left $ InvalidJWT resolvedBS
-          Right jwt -> Right (JWT.contentOf (decodeUtf8Lenient resolvedBS), jwt)
+          Right (UCAN.contentOf (decodeUtf8Lenient resolvedBS))
```

Comment:
Here you can see an effect of one of the changes.
I've changed the `UCAN.Resolver` class to only resolve to `UCAN.RawContent`, instead of also decoding the JWT.

Reply:
Hmm I have some vague memories of also going down this path (several times?) and then putting the JWT back in 🤔 I can't remember what the scenario was — something about needing the decoded case in various code paths? — so not going to say to not do this, but just mentioning it

Reply:
Okay. I just now realized that `RawContent` isn't the *whole* JWT encoded. It's only the part without the signature. That's what I get for not reading its docs!
I'll fix this in another PR. I'm thinking that the easiest way is to return either `ByteString` from `resolve`, or some kind of newtype?